buildtools/sample-server/main.js

```diff
@@ -0,0 +1,73 @@
+/* global OktaSignIn */
+/* eslint no-console: 0 */
+
+import widgetRc from '../../.widgetrc';
+
+const defaultOptions = {
+  baseUrl: 'http://rain.okta1.com:1802',
+  logo: '/img/logo_widgico.png',
+  logoText: 'Windico',
+  features: {
+    router: true,
+    rememberMe: true,
+    multiOptionalFactorEnroll: true
+  },
+  // Host the assets (i.e. jsonp files) locally
+  assets: {
+    baseUrl: '/'
+  }
+};
+
+const options = Object.assign({}, defaultOptions, widgetRc.widgetOptions);
```

Comment:
nit: Can we move the above logic out into another file?

To me, this file acts as the blueprint for rendering the Widget, where the code in this file should be nearly identical to our README, quickstarts, and sample apps. Adding configuration defaults above  _might_ confuse devs when trying to decide **where** to put their Widget config.

Reply:
Isn't this closer to what a developer would actually write though? I.e. bootstrap the widget with some options, and then call render. In the easy cases not sure how many other developers would split out widget config to a separate file.

Reply:
- maybe the `defaultOptions` is not best naming but just `widgetOptions` 

```
Object.assign({}, widgetOptions, widgetRc.widgetOptions)
```

- another approach is to move all those `defaults` to `webpack.sample.server.config` where generates those into default `widgetrc`.

Reply:
+1 on bringing defaults into the `widgetrc` file.

The case I'm trying to avoid is having developers update the `defaultOptions` instead of the `.widgetrc` file. We saw this a few times within our `oidc-js` repo, and needed to bring the default logic into a shared config to reduce friction: [`.oidc-config.js`](https://github.com/okta/okta-oidc-js/blob/master/.oidc.config.js).

Reply:
Yeah, I suppose there's no reason _not_ to have a default .widgetrc already in place, with the pre-configured defaults. Would simplify this build logic and also simplify conceptually - just change the one and it'll show up.